about.html

```diff
@@ -12,62 +13,105 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="container about-page">
+    <header>
+      <nav class="navbar">
+        <div class="nav-logo">
+          <img src="img/lambda-black.png" alt="lambda logo">
+        </div>
+        <div class="nav-links">
+          <a href="index.html">home</a>
+          <a href="about.html">about</a>
+          <a href="#">products</a>
+          <a href="#">blog</a>
+          <a href="#">contact</a>
+        </div>
+      </nav>
+      <div class="header-img">
+        <img src="img/jumbo-about.png" alt="header image">
+      </div>
+    </header>
+
+    <section class="about-top-content">
```

Comment:
You could have used the `about-page` class assigned to your outer container, instead of assigning a brand new class.
It usually a good idea to do it, it leads you to DRYer and cleaner code.